src/items/class-features/fortified_imagination_(su).json

```diff
@@ -0,0 +1,136 @@
+{
+  "_id": "O7jG2sX6am0Pl1S9",
+  "name": "Fortified Imagination (Su)",
+  "type": "feat",
+  "img": "icons/svg/mystery-man.svg",
+  "system": {
+    "type": "",
+    "ability": null,
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "",
+      "condition": "",
+      "cost": 0
+    },
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "",
+      "value": null
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.udtZxolGt5mMq9SM]{Dreamer}</p>\n<p>You’ve experienced countless nightmares, preparing you to handle anything. You gain a +2 insight bonus to @Check[type:will] saves against mind-affecting spells and abilities. As a swift action, you can spend 1 Resolve Point to grant this bonus to an ally who can perceive you.</p>"
```

Comment:
The `@Check` here is not needed; it is simply mentioning Will saves, not calling for one